backend/tests/test_error_significance_classifier.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import config
+from classifiers.error_significance_classifier import get_error_significance_score
+from random import randint
+
+
+class TestErrorSignificance(unittest.TestCase):
+    def test_get_error_significance_score_after_date_significance_limit(self):
+        for time_period in config.allowed_time_periods:
+            period_with_error_significance = config.error_significance_by_time_period[time_period]
+            distance_in_the_future = period_with_error_significance
+            self.assertEqual(get_error_significance_score(1, 1, distance_in_the_future, time_period), -1)
+
+    def test_get_error_significance_score_with_day_in_the_past(self):
+        for time_period in config.allowed_time_periods:
+            self.assertEqual(get_error_significance_score(1, 1, -1, time_period), -1)
+
+    def test_error_significance_takes_absolute_error_value(self):
+        for time_period in config.allowed_time_periods:
+            value1 = randint(1, 10)
```

Comment:
Non-deterministic tests can be troublesome when they fail and can't be reproduced... Maybe at least set the seed to a constant value at the beginning of each test-case to make them determinnistic.

Reply:
done